pkg/model/dev.go

```diff
@@ -202,6 +203,12 @@ type PersistentVolumeInfo struct {
 	Size         string `json:"size,omitempty" yaml:"size,omitempty"`
 }
 
+// InitContainer represents the initial container
+type InitContainer struct {
```

Comment:
What about also specifying the user/group? Or will this inherit the one from the main config?

Reply:
No need to specify the user group